docs/build/group-chat.md

```diff
@@ -36,15 +36,19 @@ Here are the actions:
 
 As a result of either of these actions, the user will lose access to the local database for the app installation, which includes all group chat messages they sent using the installation of your app on their device.
 
-As an app developer, this concept is important to understand and communicate to your users. For example, you might consider using this language:
-
-> If you log out of &lt;app name&gt; and log into a different app on this device, or delete &lt;app name&gt; from this device, you will lose access to all group chat messages you sent using this installation of &lt;app name&gt; on this device.
+:::tip
+To prevent your users from entering this undesirable state, [Enable message history for group chat](/docs/build/message-history).
+:::
 
-To enable your users to avoid losing access to their local databases, allow them to store their local cache in iCloud or Google Cloud, for example. This option will enable message persistence within a single app ecosystem. 
+If you choose to not enable message history, you can help users avoid losing access to their local databases by allowing them to store their local cache in iCloud or Google Cloud, for example. This option will enable message persistence within a single app ecosystem.
```

Comment:
But we still advice at the very least adding a url on create so other apps may benefit from message history.